src/styles/styles.js

```diff
@@ -2179,6 +2179,22 @@ const webViewStyles = {
     },
 };
 
+/**
+ * Return typing indicator text style
+ * @param {Boolean} isSmallScreenWidth
+ *
+ * @returns {Object}
+ */
+function getTypingIndicatorTextStyle(isSmallScreenWidth) {
+    return isSmallScreenWidth
+        ? {
+            maxWidth: 200,
+        }
+        : {
+            maxWidth: 500,
+        };
+}
+
```

Comment:
Hmm. This is supposed to be fluid. Take max available space with hiding `is typing...`